docs/discordsdk.rst

```diff
@@ -0,0 +1,157 @@
+discordsdk package
+==================
+
+Submodules
+----------
+
+discordsdk.achievement module
+-----------------------------
+
+.. automodule:: discordsdk.achievement
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.activity module
+--------------------------
+
+.. automodule:: discordsdk.activity
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.application module
+-----------------------------
+
+.. automodule:: discordsdk.application
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.discord module
+-------------------------
+
+.. automodule:: discordsdk.discord
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.enum module
+----------------------
+
+.. automodule:: discordsdk.enum
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+discordsdk.event module
+-----------------------
```

Comment:
This module is only used internally, so we might not need to put this on the auto-docs.

Reply:
done!